pyspecProcScripts/correlation_alignment.py

```diff
@@ -401,4 +368,12 @@ def correl_align(
         plt.gca().yaxis.set_major_formatter(to_percent)
     if fl is not None:
         fl.pop_marker()
+    # {{{ Make sure returned f_shift is the same ndshape that the
+    #     data was originally fed in - meaning we must chunk it back
+    #     into the original repeat_dims or rename back to the original
+    #     indirect name
+    if len(indirect) > 1:
+        f_shift.chunk("repeats", indirect)
+    else:
+        f_shift.rename("repeats", indirect[0])
```

Comment:
this is very good, but is missing a `# }}}`

same comment about the change of name (indirect vs. repeat_dims) as above